lib/interceptor/cache.js

```diff
@@ -84,15 +87,38 @@ class CacheHandler extends DecoratorHandler {
       }
     }
     return this.#handler.onData(chunk)
```

Comment:
This forwards the data to the next handler. What that handler is doesn't really matter in this case.